modules/custom/express_site_metrics/express_site_metrics.module

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/**
+ * @file
+ * Provides statistics reporting for Express web sites.
+ */
+
+$tokenHandlerConstructor = [
+  'metricsUsers' => 'express_site_metrics_users',
+  'secret' => 'express_site_metrics_secret',
+  'dbGet' => 'variable_get',
+  'dbSet' => 'variable_set',
+];
+
+/**
+ * Implements hook_menu().
+ */
+function express_site_metrics_menu() {
```

Comment:
Consider putting this in `/admin/config`. The other items that require this kind of access level are in there. I would remove it from the top level admin menu

Reply:
so the url/path would be `/admin/config/express/metrics`?

Reply:
That seems reasonable to me